plugin.yml

```
@@ -1,9 +1,14 @@
 name: SimpleCape
-api: [3.0.0]
 main: padreon\SimpleCape\Main
+website: https://github.com/padreon/SimpleCape
+description: This is a SimpleCape plugin for Servers
+api: 3.0.0
```

```suggestion
api: [3.0.0]
```
is like xenial said,  keep this [ ]